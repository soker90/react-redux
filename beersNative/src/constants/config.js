let config = null

if (process.env.NODE_ENV === 'development') {
  config = require('./config.dev').default
} else {
  config = require('./config.pro').default
}

export default config
