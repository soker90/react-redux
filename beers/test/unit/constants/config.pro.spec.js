// config.pro.spec.js
import config from '../../../src/constants/config.pro'

describe('Config PRO constants', function () {
  test('exist', () => {
    expect(config).toMatchSnapshot()
  })
})
