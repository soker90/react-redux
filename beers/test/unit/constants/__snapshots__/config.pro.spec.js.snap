exports[`Config PRO constants exist 1`] = `
Object {
  "API_URL": "http://myservicesdomain.com/api",
}
`;
